src/calculateRentalCost.js

```diff
@@ -29,6 +29,13 @@
 
 function calculateRentalCost(days) {
   // write code here
+  if (days >= 7) {
+    return (days * 40) - 50;
+  } else if (days >= 3 && days <= 7) {
```

Comment:
```suggestion
  } else if (days >= 3) {
```
Якщо дійде до цієї перевірки, значить не пройшла перша, а отже днів точно буде <=7

Reply:
Я не розумію я їх створити. Сама функція пише постійно days is not declared
![изображение](https://user-images.githubusercontent.com/59259022/142635538-989e9ce6-02a2-4971-bbbb-ac45361a4c1c.png)